src/deploy.sh

```diff
@@ -552,7 +552,8 @@ function prepare_remote_dir()
   # TODO: In some point, we need to move the below code to ArchLinux specific
   # file
   if [[ "$distro" == 'arch' ]]; then
-    cp2remote "$flag" "$KW_ETC_DIR/template_mkinitcpio.preset" "$REMOTE_KW_DEPLOY"
+    cp2remote "$flag" "${KW_ETC_DIR}/template_mkinitcpio.preset" "$REMOTE_KW_DEPLOY"
+    cp2remote "$flag" "${KW_ETC_DIR}/template_loader.conf" "$REMOTE_KW_DEPLOY"
```

Comment:
This ends up copying the systemd-boot loader template to arch installations that use GRUB as well.